python/lang/correctness/return-in-init.py

```diff
@@ -75,3 +75,41 @@ class H:
     def __init__(self, x):
         # ok:return-in-init
         return None
+
+class Odd:
+    def __init__(self, numbers):
+        def is_odd(n):
+            # ok:return-in-init
+            return n % 2 == 1
+        self.numbers = filter(is_odd, numbers)
+
+        # ruleid:return-in-init
```

Comment:
I guess this would be a `todoruleid` because the new rules won't find any matches if you define a nested function inside `__init__`.

Reply:
:+1: I changed to `todoruleid`